dv/cs_registers/reg_driver/csr_listing.def

```diff
@@ -0,0 +1,32 @@
+// Copyright lowRISC contributors.
```

Comment:
Making this file `.h` could make it easier for build tools.

Reply:
What do you think @lenary ?

Reply:
I think it's useful to have a differentiator between 'headers' -- which contain declarations, and are only included once (or at least, only have an effect the first time they're included ) -- and this kind of data definition -- which will have a different effect every time its included. The file extension seems the best place to make this difference clear. I hope it is relatively easy to teach build tools that a `.def` file might still be a dependency of a compilation unit.